### ISSUE TITLE
hypospermia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9961]
**Preferred term label:**

**Synonyms**
Low volume of ejaculated semen

**Definition (free text, please give PubMed ID)**
Sperm volume is considered normal when it ranges between 1.5 and 6 ml according to the reference values published by the World Health Organization (WHO)

PMID: 18801689 Hypospermia is a semen volume lower than 2 mL on at least two semen analyses.

PMID: 32025909 In case of CBAVD associated with bilateral absence of SV, the biological warning sign is a non-pathognomonic 
triad: hypospermia (semen volume < 1.5  ml), acid pH (<7.0) and levels of seminal plasma biochemical markers 
lower than the lower references values (fructose<13 μmol/ejaculate; GPC<2 µmol/ejaculate)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Partial retrograde ejaculation
Vasa deferentia and seminal vesicle agenesis
Androgen Insensitivity Syndrome ORPHA:754 
Congenital bilateral absence of vas deferens ORPHA:48

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
This is [Low semen volume HP:6000135](https://hpo.jax.org/browse/term/HP:6000135)
